### ISSUE TITLE
Revert "Improve suggested candidate filters in element picker"

### ISSUE THREAD

**Issue by Yuki2718** [uBlockOrigin/uBlock-issues#3220]
### Prerequisites

- [X] I verified that this is not a filter list issue. Report any issues with filter lists or broken website functionality in the [uAssets issue tracker](https://github.com/uBlockOrigin/uAssets/issues).
- [X] This is **NOT** a [_YouTube_](https://www.github.com/uBlockOrigin/uAssets/issues/20586), [_Facebook_](https://www.github.com/uBlockOrigin/uAssets/issues/3367) or [_Twitch_](https://www.github.com/uBlockOrigin/uAssets/issues/5184) report. _These sites MUST be reported by clicking their **respective links**_.
- [X] This is not a support issue or a question. For support, questions, or help, visit [/r/uBlockOrigin](https://www.reddit.com/r/uBlockOrigin/).
- [X] I performed a [cursory search of the issue tracker](https://github.com/uBlockOrigin/uBlock-issues/issues?q=is%3Aissue) to avoid opening a duplicate issue.
- [X] The issue is not present after disabling uBO in the browser.
- [X] I checked the [documentation](https://github.com/gorhill/uBlock/wiki) to understand that the issue I am reporting is not normal behavior.

### I tried to reproduce the issue when...

- [X] uBO is the only extension.
- [X] uBO uses default lists and settings.
- [X] using a new, unmodified browser profile.

### Description

Explained in https://github.com/uBlockOrigin/uBlock-discussions/discussions/889 but I'm opening an issue here hopefully to gather more opinion.

The change was made in https://github.com/gorhill/uBlock/commit/c0b56cffabfc08bf7f4ab172be8b8f8b1a878044#diff-1d458db50255dc41294f582831b2dd40eb14ac9e636ba1a8e94b7163982a1e18R898-R911 and now the picker is sometimes useless - unable to choose ID or full href on many sites.

https://github.com/uBlockOrigin/uBlock-issues/assets/58900598/dc2d6983-88a7-42d5-b4d6-5f9b8ef2d101

### A specific URL where the issue occurs.

```yaml
https://www.comnico.jp/we-love-social/sns-users
```

### Steps to Reproduce

1. Visit the page and scroll down until the popup is shown
2. Choose the selector having ID
3. Move the slider around, you'll never be able to choose ID

### Expected behavior

Previous one

### Actual behavior

Explained

### uBO version

1.57.3b5

### Browser name and version

Chrome 123.0.6312.124

### Operating System and version

Windows 10

**Comment by Yuki2718** [comment]
Unrelated but not something worth another issue, but I feel the height of the filters window is now too low. It should be higher or adjustable.

**Comment by gorhill** [comment]
> filters window

Are you talking about the text area of the list of candidates?

**Comment by garry-ut99** [comment]
> I feel the height of the filters window in the picker is now too low.

Worth to provide the whole  context, it is a side-effect of the whole Element Picker's height reduction on Mobile: https://github.com/gorhill/uBlock/commit/953c978d59  - which was necessary, and since Element Picker code is same/shared between Desktop & Mobile for code simplicity and easier maintentance, as a side-effect the Picker's height on Desktop was reduced accordingly/respectively. 

> Unrelated but not something worth another issue

A thread already exists - the aforementioned commit was related to this discussion: 
- https://github.com/uBlockOrigin/uBlock-issues/discussions/1929

**Comment by Yuki2718** [comment]
> Are you talking about the text area of the list of candidates?

Yes.

> which was necessary

Even for mobile it's too low IMO.

**Comment by gorhill** [comment]
My question had a typo, which makes me unsure about your answer. Which part is not high enough? The text area **or** the list of candidates?

**Comment by Yuki2718** [comment]
TBH both of them, but in particular the text area.

**Comment by freezer2022** [comment]
I agree with Yuki2718, text area height is now often too low (inconvenient and cumbersome) when there is a need to view/edit/manage some very long filters, which can take more than 4 rows, and they just don't fit in the text area. Making the Picker window resizable or adding incerase height button would be a good idea.
